src/widgets/activity-group/model/factories/ScheduledGroupEvaluator.test.ts

```diff
@@ -28,6 +28,7 @@ const getProgress = (startAt: Date, endAt: Date | null) => {
           startAt,
           endAt,
           availableTo: null,
+          entityName: 'mock-entity-name',
```

Comment:
Change it to empty string, please. The related BL doesn't use this field. And if even so, it'd be easy to catch.